packages/components/pages/f-mfa/src/components/Mfa.vue

```diff
@@ -43,11 +153,41 @@ export default {
 .c-mfa {
     display: flex;
     justify-content: center;
-    min-height: 80vh;
-    width: 80vw;
     margin: auto;
-    border: 1px solid red;
-    font-family: f.$font-family-base;
-    @include f.font-size(heading-m);
+    text-align: center;
+
+    @include f.media('>=narrow') {
+        width: 80vw;
+    }
+
+    h2 {
```

Comment:
I would add a class to the `h2` (`c-mfa-card`) and add a selector based on that. JUst means if the tag ever changes in the future for some reason, the CSS won't break